docs/dictionary/function/revOpenDatabase.lcdoc

```diff
@@ -3,15 +3,15 @@ Name: revOpenDatabase
 Synonyms: revdb_connect
 
 Type: function
-Syntax: revOpenDatabase(<databaseType>, {<hostAddress>, <databaseName>, [<userName>],[<passWord>],{[<useSSL>], [<socketAddr>], [<rwTimeout>], [<autoReconnect>] | [<cursorType>] | [<sslOption>],...} | <filePath>,[<sqliteOptions>]})
+Syntax: revOpenDatabase(<databaseType>, {<hostAddress>, <databaseName>, [<userName>],[<passWord>],{[<useSSL>], [<socketAddr>], [<rwTimeout>], [<autoReconnect>] | [<cursorType>] | [<sslOption> [,...]]} | <filePath>,[<sqliteOptions>]})
 
 Syntax: revOpenDatabase("mysql", <hostAddress>, <databaseName>, [<userName>],[<passWord>],[<useSSL>], [<socketAddr>], [<rwTimeout>], [<autoReconnect>])
 
 Syntax: revOpenDatabase("odbc", <hostAddress>, <databaseName>, [<userName>],[<passWord>],[<cursorType>])
 
 Syntax: revOpenDatabase("sqlite",<filePath>,[<sqliteOptions>])
 
-Syntax: revOpenDatabase("postgresql", <hostAddress>, <databaseName>, [<userName>], [<passWord>], [<sslOption>],...)
+Syntax: revOpenDatabase("postgresql", <hostAddress>, <databaseName>, [<userName>], [<passWord>], [<sslOption> [,...]])
```

Comment:
We should probably be consistent with a space between `,` and `...` too

Reply:
ok, yes... it should be `[<sslOption>[ ,...]]`

Reply:
These should be:
```
revOpenDatabase("postgresql", <hostAddress>, <databaseName>, [<userName>], [<passWord>] [, <sslOption> ...])
```